src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -865,13 +861,8 @@ MulticopterAttitudeControl::run()
 			float dt = (now - last_run) / 1e6f;
 			last_run = now;
 
-			/* guard against too small (< 0.2ms) and too large (> 20ms) dt's */
-			if (dt < 0.0002f) {
-				dt = 0.0002f;
-
-			} else if (dt > 0.02f) {
-				dt = 0.02f;
-			}
+			// Guard against too small (< 0.2ms) and too large (> 20ms) dt's.
+			math::constrain(dt, 0.0002f, 0.02f);
```

Comment:
You could combine the entire thing into a single line and then make dt const.

```suggestion
			const float dt = math::constrain((now - last_run) / 1e6f, 0.0002f, 0.02f);
```

Reply:
Awesome!  Great suggestion!

Reply:
Done!